eng/Versions.props

```diff
@@ -127,6 +127,7 @@
     <SystemPrivateRuntimeUnicodeDataVersion>7.0.0-beta.21602.1</SystemPrivateRuntimeUnicodeDataVersion>
     <SystemRuntimeTimeZoneDataVersion>7.0.0-beta.21602.1</SystemRuntimeTimeZoneDataVersion>
     <SystemSecurityCryptographyX509CertificatesTestDataVersion>7.0.0-beta.21602.1</SystemSecurityCryptographyX509CertificatesTestDataVersion>
+    <SystemTextRegularExpressionsTestDataVersion>7.0.0-beta.21602.1</SystemTextRegularExpressionsTestDataVersion>
```

Comment:
I believe you also need to add a line in Version.Details.xml file in order to have this one update automatically via Darc.  You can see an example for the one above here:

[https://github.com/dotnet/runtime/blob/c9f9697fe7d10c0c1d3f17cb258518bc8d4e7cc6/eng/Version.Details.xml#L125-L128](https://github.com/dotnet/runtime/blob/c9f9697fe7d10c0c1d3f17cb258518bc8d4e7cc6/eng/Version.Details.xml#L125-L128)

Just copy and paste that and adjust the Name.

Reply:
Ah, cool, thanks.

Reply:
Given CI is almost done, it's fine if you want to address this in a subsequent PR. Or I can do that too on mine once I rebase after you merge the code cleanup one.

Reply:
It's ok.  I actually want to wait until after #62318 is merged, and then do an outerloop run with that before merging this.